### ISSUE TITLE
Feature request: Add start, pause, reset option

### ISSUE THREAD

**Issue by b3n10** [parnoldx/timer#13]
I was trying to double click on the app itself to see if it will pause or reset the timer but nothing happened. Hoping you can add this feature.

**Comment by a-tokarev** [comment]
Yes, please. Tried to just stop the timer, but instead it resets itself again and again.
